lib/manageiq/messaging/kafka/common.rb

```diff
@@ -21,12 +21,9 @@ def consumer(beginning, options)
           @consumer = kafka_client.consumer
         end
 
-        def raw_publish(wait, body, options)
+        def raw_publish(body, options)
           options[:payload] = encode_body(options[:headers], body)
-          producer.produce(options).tap do |handle|
-            handle.wait if wait
-            logger.info("Published to topic(#{options[:topic]}), msg(#{payload_log(body.inspect)})")
```

Comment:
This much logging seemed like overkill (especially to `.info`), we already set `logger` as the `RdKafka` logger.

Reply:
🔥